Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,44 @@
 import React from 'react';
 import './App.css';
 
+
+
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
+
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="AllClasses">
```

Comment:
might be better to use a className of container here, or calc-container or something, just for extra clarity to someone looking at the css for example